x/proof/types/params.go

```diff
@@ -36,7 +36,7 @@ var (
 	ParamProofSubmissionFee = "proof_submission_fee"
 	// TODO_MAINNET: Determine a sensible default value for the proof submission fee.
 	// MinProofSubmissionFee is the default and minimum fee for submitting a proof.
-	MinProofSubmissionFee = cosmostypes.NewCoin(volatile.DenomuPOKT, math.NewInt(100))
+	MinProofSubmissionFee = cosmostypes.NewCoin(volatile.DenomuPOKT, math.NewInt(0)) // TODO_IN_THIS_PR: any issues with having this as 0?
```

Comment:
This is fine, it's just indication the minimum stake gov is allowed to set.

We'll need to update it through a gov transaction though.